libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -724,11 +722,29 @@ void CodeTransform::visitExpression(Expression const& _expression)
 
 void CodeTransform::visitStatements(vector<Statement> const& _statements)
 {
+	boost::optional<AbstractAssembly::LabelID> jumpTarget = boost::make_optional(false, AbstractAssembly::LabelID());
```

Comment:
```suggestion
	boost::optional<AbstractAssembly::LabelID> jumpTarget;
```

That's not the stack problem, but still :-). (The default constructor will create an uninitialized ``optional`` as well)

Reply:
There's a boost bug; I added a link to it in a comment

Reply:
Hm... the link refers to GCC < 5.1... we recently increased the minimal version to 5.0... and we require a minimum of boost 1.65 (whereas the link is for boost 1.63), but it's in boost 1.65 as well it seems (https://www.boost.org/doc/libs/1_65_0/libs/optional/doc/html/boost_optional/tutorial/gotchas/false_positive_with__wmaybe_uninitialized.html) - so strictly speaking, I guess we might still be affected by this...

My guess would be that we do this at other places in the code as well... but yeah - I'd say that both keeping this workaround and removing it is fine, so as you like ;-)!

Reply:
@ekpyron Yeah I got a unexpected CI failure so I had to put this workaround.
Could you resolve the request changes so that this PR can be merged?